plugins/MidiImport/MidiImport.cpp

```diff
@@ -467,7 +467,7 @@ bool MidiImport::readSMF( TrackContainer* tc )
 				if( update == "programi" )
 				{
 					long prog = evt->get_integer_value();
-					if( ch->isSF2 )
+					if( ch->isSoundFont )
```

Comment:
(Non-blocking) Per the Coding Conventions [re: parentheses](https://github.com/LMMS/lmms/wiki/Coding-Conventions#braces), [re: flow control](https://github.com/LMMS/lmms/wiki/Coding-Conventions#flow-control-statements):

> 1. Spaces MUST NOT be added after an opening parenthesis or bracket
> 2. Spaces MUST NOT be added before a closing parenthesis or bracket

and,

> Flow control statements (if, else if, else, for, do, while, and switch) MUST have a space between the keyword and the opening parenthesis.

```suggestion
					if (ch->isSoundFont)
```